src/dbnode/storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -884,6 +927,27 @@ func (s *fileSystemSource) bootstrapFromIndexPersistedBlocks(
 	return res, nil
 }
 
+func (s *fileSystemSource) loadInfoFiles(
+	namespaces bootstrap.Namespaces,
+) map[namespace.Metadata]fs.InfoFileResultsPerShard {
```

Comment:
If we move up caching we could move this function to the `bootstrapper` package.